.appveyor.yml

```diff
@@ -1,8 +1,6 @@
 version: '{build}'
 skip_tags: true
-
-install:
-    - cinst jdk10 -params 'installdir=C:\\jdk10'
+skip_branch_with_pr: true
```

Comment:
How does this change CI behavior?

Reply:
This builds and tests all of Trellis on Windows machines. The `skip_branch_with_pr` means that it doesn't build the same thing twice. Removing the jdk10 tests is temporary until the JDK installation process starts working again -- a new JDK 10 version was released yesterday, and the installer packages haven't caught up, causing all the builds to fail, so I'm just removing Windows-based JDK10 tests for now.